public/index.html

```diff
@@ -1,33 +1,30 @@
 <html>
 
 <head>
-  <title>A-Frame Social VR Demo</title>
-  <script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/1.4.5/socket.io.min.js"></script>
-  
-  <script src="easyrtc/easyrtc.js"></script>
-  <script src="./app.bundle.js"></script>
+    <title>A-Frame Social VR Demo</title>
+    <script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
+    <script src="./app.bundle.js"></script>
 </head>
 
 <body>
-  <a-scene
-    networked-scene="adapter: easyrtc;
+    <a-scene networked-scene="adapter: janus;
+                     room: 2;
+                     serverURL: wss://quander.me:8989;
```

Comment:
Is all of us hitting this server going to cause any issues with any testing you guys are doing?

Reply:
I put this demo on another room id. So we'll be separated from all that.